lib/oban/live_dashboard.ex

```diff
@@ -79,8 +71,29 @@ defmodule Oban.LiveDashboard do
     """
   end
 
+  @impl true
+  def mount(params, _, socket) do
+    socket =
+      socket
+      |> assign(job_state: Map.get(params, "job_state", "executing"))
+      |> assign(sort_by: Map.get(params, "job_state"))
```

Comment:
This should be moved to `handle_params` otherwise `sort_by` and `job_state` won't get updated except for when the view is re-mounted.

Reply:
Done. `handle_params` deserves a refactor later IMO.